string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,15 @@
+/* exported capitalizeWord */
+// make storage for the capitalized word
+// uppercase the first letter, add the rest of the letters but lowercased and store.
+// if the 4th letter of the word is 's',
+//   make the word JavaScript
+// return location of storage for capitalized word
+
+var capitalWord = '';
```

Comment:
You should never create a variable meant to be used only locally in a function outside that function block. You should move this inside the function definition.